### PR TITLE
Add tests for utils_common to improve coverage

### PR DESCRIPTION
**Pull request by staniewzki**
### Checklist

- [x] Code compiles without errors locally
- [x] All tests pass locally
- [x] CI workflows execute properly

- [x] CI workflows, not executed per PR (e.g. Nightly), execute properly 
- [x] All newly added source files have a license
- [x] All newly added source files are referenced in CMake files

### CODE REVIEW

File: test/utils/utils.cpp [hunk 1]
```diff
@@ -57,3 +57,48 @@ TEST_F(test, utils_parse_var) {
     EXPECT_TRUE(utils_parse_var("test1;test2;test3,abc;test4", "test3", &arg));
     EXPECT_TRUE(utils_parse_var("test1;test2;test3;test4,abc", "test4", &arg));
 }
+
+TEST_F(test, utils_translate_mem_visibility_flag) {
+    umf_memory_visibility_t in_flag = static_cast<umf_memory_visibility_t>(0);
+    unsigned out_flag;
+    auto ret = utils_translate_mem_visibility_flag(in_flag, &out_flag);
+    EXPECT_EQ(ret, UMF_RESULT_ERROR_INVALID_ARGUMENT);
```
Comment (lukaszstolarczuk) [inline test/utils/utils.cpp:65]: ASSERT_EQ ?

File: test/utils/utils.cpp [hunk 2]
```diff
@@ -57,3 +57,48 @@ TEST_F(test, utils_parse_var) {
...
+    auto ret = utils_get_file_size(-1, &size);
+    ASSERT_EQ(ret, -1);
+
+    int fd = utils_create_anonymous_fd();
+    ret = utils_get_file_size(fd, &size);
+    ASSERT_EQ(ret, 0);
+    ASSERT_EQ(size, 0);
+}
+
+TEST_F(test, utils_set_file_size) {
```
Comment (lukaszstolarczuk) [inline test/utils/utils.cpp:87]: usually we add in the name information that the test is negatve, e.g. here: `utils_set_wrong_file_size` - pls apply to all new test cases

**Review by lukaszstolarczuk — COMMENTED** [review]
when you're ready for merge, pls squash your changes into a single commit

**Comment by bratpiorka** [comment]
note - please keep commits squashed unless you want to intentionally split the work

File: test/utils/utils_linux.cpp [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    ret = utils_shm_create("", 256);
+    EXPECT_EQ(ret, -1);
+
+    ret = utils_shm_create("/abc", 256);
+    EXPECT_GE(ret, 0);
+
+    ret = utils_shm_create("/abc", -1);
+    EXPECT_EQ(ret, -1);
+}
```
Comment (lukaszstolarczuk) [inline test/utils/utils_linux.cpp:52]: missing EOF

**Comment by bratpiorka** [comment]
how the coverage changes with this PR?

**Comment by lukaszstolarczuk** [comment]
> how the coverage changes with this PR?

81,3 -> 81,6

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**